components/scheduler/events.md

```diff
@@ -939,6 +940,106 @@ Through its event arguments you can get the `Item` to cast it to your model type
 </style>
 ````
 
+## OnCellRender
+
+The `OnCellRender` event fires upon rendering of each slot cell. It allows you set custom CSS class to the cells based on the business logic.
+
+The handler receives an argument of type `SchedulerCellRenderEventArgs` which exposes the following fields:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Field | Type | Description |
+| --- | --- | --- |
+| `Class` | `string` | The CSS class that will be applied to the cells. The CSS rules that are set for that class will be visibly rendered on the Scheduler cells. |
```

Comment:
The second sentence is self-evident, and at the same time, it guarantees something that is not that certain :)

```suggestion
| `Class` | `string` | The CSS class that will be applied to the cell. |
```